tests/unit/hmrc/mtdFraudPrevention.test.js

```diff
@@ -100,4 +100,4 @@ describe("FraudPreventionHeaders", () => {
     expect(headers.get("Gov-Client-Device-ID")).toEqual("fce4f7ff-d5f1-4e4f-99a1-aa97bef71e99");
     expect(errors[0]).toEqual("Something went wrong.");
   });
-});
+});
```

Comment:
This file shouldn't have changed - please checkout the same version from master - here is a guide http://nicolasgallagher.com/git-checkout-specific-files-from-another-branch/

Reply:
Cleaned. this up, refer new PR https://github.com/intuit/user-data-for-fraud-prevention/pull/67